src/calculateRentalCost.js

```diff
@@ -5,6 +5,17 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const basePrice = days * 40;
+
+  if (days >= 7) {
+    return basePrice - 50;
+  }
+
+  if (days >= 3) {
+    return basePrice - 20;
+  }
+
+  return basePrice;
```

Comment:
The function name `calculateRentalCost` is descriptive enough, but the magic numbers within the function should be replaced with named constants to improve readability and maintainability of the code.